common/src/commonMain/kotlin/com/cuhacking/atlas/common/FeatureApi.kt

```diff
@@ -21,10 +23,13 @@ class FeatureApi(private val database: Database, private val client: HttpClient)
         this)
 
     suspend fun getAndStoreFeatures() = withContext(CoroutineDispatchers.io) {
-        val features: List<Feature> = client.get<String>(AtlasConfig.SERVER_URL).toFeatureCollection().features
+        val response = client.get<String>(AtlasConfig.SERVER_URL)
```

Comment:
Be careful to handle possible network errors that could happen here. It probably needs to be wrapped in a try/catch.